resources/styles/main.scss

```diff
@@ -1,6 +1,7 @@
 @use "config";
-@use "common";
+@use "common" as *;
 @use "utilities";
+@use "config/colors" as colors;
```

Comment:
This doesn't have anything to do with the BpTabs component, but you shouldn't need to `@use "config/colors"` here since `/colors` is already coming in on Line 1 as part of `@use "config"`.